satellite/metainfo/metainfo.go

```diff
@@ -645,15 +645,10 @@ func (endpoint *Endpoint) CreateBucket(ctx context.Context, req *pb.BucketCreate
 		return nil, status.Errorf(codes.InvalidArgument, err.Error())
 	}
 
-	// checks if bucket exists before updates it or makes a new entry
+	// checks if bucket exists before updates it with partner id if it exits or makes a new entry
+	partnerID := keyInfo.PartnerID
 	bucket, err := endpoint.metainfo.GetBucket(ctx, req.GetName(), keyInfo.ProjectID)
 	if err == nil {
-		var partnerID uuid.UUID
```

Comment:
Do we still need this logic for the app-connector?

Reply:
updated the logic